### PR TITLE
Use variable to calculate version

### PR DESCRIPTION
**Pull request by carrodher**
**Use $JENKINS_WAR variable**
```diff
-        version="$(java -jar /usr/share/jenkins/jenkins.war --version)"
+        version="$(java -jar $JENKINS_WAR --version)"
```

### CODE REVIEW

**Comment by carlossg** [comment]
tests are failing because now install-plugins refuses to install the plugins already in detached-plugins folder. I'm not sure if those plugins are used or install-plugins should install them again the the plugins dir

**Comment by carrodher** [comment]
I have been rechecking the changes in the script and it seems that these changes are not related to the failures in the tests. Should we leave this PR isolated with these changes and open an issue/fix the original problem?

**Comment by carlossg** [comment]
I think they are because now the script is checking if the plugin is already installed and finds it under `WEB-INF/detached-plugins`

**Comment by carrodher** [comment]
@carlossg what do you think about new changes?

File: install-plugins.sh [hunk 1]
```diff
@@ -166,7 +166,6 @@ versionFromPlugin() {
     else
         echo "latest"
     fi
-
```
Comment (batmat) [inline install-plugins.sh]: side note: generally avoid staging unrelated changes. This creates a more complicated history and makes code archeology harder to dig into.

**Comment by carrodher** [comment]
I have undone the unrelated changes
